sources/api/controllers/UserController.ts

```diff
@@ -92,6 +93,12 @@ export async function confirmUserEmail(req: Request, res: Response) {
     await conn.manager.save(user);
     await conn.manager.remove(pendingUser);
 
+    const calendar = new Calendar("Main Calendar", [], []);
+    const createdCalendar = await conn.manager.save(calendar);
+
+    const calendarsToOwners = new CalendarsToOwners(createdCalendar.id, user.id);
+    await conn.manager.save(calendarsToOwners);
```

Comment:
Will change with https://github.com/PerfectWeek/web-api/pull/106

FYI:
```ts
Calendar.create(conn, calendar, [requesting_user]);
```

Also, I was thinking about removing the two lists in the constructor since we only use it with empty brackets `[]`

What do you think ?

Reply:
I have to admit i just put in thinking it could be possible to initialize a Calendar with multiple owners, but i don't think it would be useful since we admitted a group can only have one calendar